resources/referers.yml

```diff
@@ -3377,6 +3383,12 @@ search:
       - ricercanews.virgilio.it
       - mobile.virgilio.it
 
+  Vi-view
```

Comment:
There's a whole host of these search engines under "inspsearch.com" - I classified this as "Inspsearch" for the whole domain.

Not sure what we should call this "provider"